stream-chat-android-compose-sample/src/main/java/io/getstream/chat/android/compose/sample/ui/login/UserLoginActivity.kt

```diff
@@ -78,10 +78,8 @@ class UserLoginActivity : AppCompatActivity() {
                             // login screen then we need to reinitialize the SDK with our API key.
                             ChatHelper.initializeSdk(applicationContext, userCredentials.apiKey)
                         }
-                        ChatHelper.connectUser(
-                            userCredentials = userCredentials,
-                            onSuccess = ::openChannels,
-                        )
+                        ChatHelper.connectUser(userCredentials = userCredentials)
+                        openChannels()
```

Comment:
Hey @adasiewiczr,

Shouldn't we wait for the successful result before opening Channels?

Reply:
not in the samples - as we want ot be able to log in offline

Reply:
we won't get success if we are offline. We could either set the timeout, but then we'll have to block UI and don't allow clicking multiple times, or move forward without checking the result.
This behavior was aligned with XML, we might changed it in the future

Reply:
On our sample app it is safe because we are using static token that should be valid always without any expiration time neither become invalid at any point. So, as far as we connect the user, the SDK will handle all needed reconnection when the device is online again.